src/main/java/org/kuali/kfs/kim/document/rule/AttributeValidationHelper.java

```diff
@@ -47,14 +47,14 @@
  * CU Customization (KFSPTS-23531):
  * 
  * Updated the generation of the cache keys for improved compatibility with Redis.
+ * Backported redis fix on FINP-8169
  */
 public class AttributeValidationHelper {
 
     private static final Logger LOG = LogManager.getLogger();
     private static final String DOCUMENT_PROPERTY_PREFIX = KRADConstants.DOCUMENT_PROPERTY_NAME + ".";
 
     // CU Customization (KFSPTS-23531): Add MessageFormat-related constants for generating various cache keys.
-    private static final String ATTRIBUTE_BY_ID_CACHE_KEY_PATTERN = "'{getAttributeDefinitionById}'-id={0}";
     private static final String ATTRIBUTE_BY_NAME_CACHE_KEY_PATTERN = "'{getAttributeDefinitionByName}'-name={0}";
```

Comment:
I'm fine with keeping this custom cache key in place until we're ready to backport KualiCo's follow-up FINP-8210 fixes.